colors/lighthaus.vim

```diff
@@ -346,6 +346,15 @@ call s:h("sqlKeyword",    s:cyan2,      "",   "")
 call s:h("sqlSpecial",    s:orange2,    "",   "")
 " }
 
+" ― ― ― ― ― ― ― ― ― 
+" ―  VIMWIKI ― 
+" ― ― ― ― ― ― ― ― ― 
+call s:h("VimwikiHeader1",   s:fg_alt,  "",   s:B)
+call s:h("VimwikiHeader2",   s:green,   "",   s:B)
+call s:h("VimwikiHeader3",   s:orange,  "",   s:B)
+call s:h("VimwikiHeader4",   s:purple,  "",   s:B)
+call s:h("VimwikiHeader5",   s:cyan,    "",   s:B)
+call s:h("VimwikiHeader6",   s:blue,    "",   s:B)
```

Comment:
`s:white`

Header6 is fairly low in terms of priority, doesn't need to pop nearly as much. `s:white` pops just the right amount to be spotted and yet not be right in your face I felt.